Just disable inclusion of fmemopen since Makefile does not offer CFLAGS macro.
--- a/main.c	2021-07-01 11:56:11.657844919 +0200
+++ b/main.c	2021-07-01 11:56:18.733756580 +0200
@@ -10,11 +10,6 @@
 #include "msd/bootcode4.h"
 #include "msd/start4.h"
 
-/* Assume BSD without native fmemopen() if doesn't seem to be glibc */
-#if defined(__APPLE__) || (!defined(_GNU_SOURCE) && (!defined(_POSIX_C_SOURCE) || _POSIX_C_SOURCE < 200809L))
-#include "fmemopen.c" // BSD fmemopen() compat in terms of funopen()
-#endif
-
 int signed_boot = 0;
 int verbose = 0;
 int loop = 0;
